VenafiPS/Public/New-TppCustomField.ps1

```diff
@@ -0,0 +1,221 @@
+<#
+.SYNOPSIS
+Create new custom field in Venafi of the specified class and type.
+
+.DESCRIPTION
+Creates a new custom field in Venafi. Required parameters are name, label, class and type.
+See Venafi docs for information about other metadata items.
+
+.PARAMETER Name
+Name of new custom field
+
+.PARAMETER Label
+Label of new custom field
+
+.PARAMETER Class
+Class of new custom field, either 'Device' or 'X509 Certificate'
+
+.PARAMETER Type
+Type of new custom field
+
+.PARAMETER VenafiSession
+Session object created from New-VenafiSession method.  The value defaults to the script session object $VenafiSession.
+
+.INPUTS
+None
+
+.OUTPUTS
+Query returns a PSCustomObject with the following properties:
+    AllowedValues
+    Class
+    DateOnly
+    DN
+    ErrorMessage
+    Guid
+    Help
+    Label
+    Mandatory
+    Name
+    Policyable
+    RegularExpression
+    RenderHidden
+    RenderReadOnly
+    Single
+    TimeOnly
+    Type
+
+.EXAMPLE
+New-TppCustomField -Name "Last Date" -Label "Last Date" -Class "X509 Certificate" -Type "1" -RenderReadOnly -Help "Last Date of certificate import"
```

Comment:
now that Type changed, just need to update it here and we're all good

Reply:
Fixed